lib/ui/product/widget/product_description.dart

```diff
@@ -0,0 +1,336 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_icons/flutter_icons.dart';
+import 'package:google_fonts/google_fonts.dart';
+import 'package:flipper/utils/HexColor.dart';
+
+class ProductDescription extends StatefulWidget {
+  const ProductDescription({Key key}) : super(key: key);
+
+  @override
+  _ProductDescriptionState createState() => _ProductDescriptionState();
+}
+
+class _ProductDescriptionState extends State<ProductDescription> {
+  double _checked;
+  TextEditingController quantityController = new TextEditingController();
+  TextEditingController notesController = new TextEditingController();
+  double _slider = 0;
+  int quantity = 1;
+  double paramsDefault = 230.8;
+  double titleAmount = 0;
+  @override
+  void initState() {
+    super.initState();
+  }
+
+  void _increaseQty() {
+    setState(() {
+      quantity++;
+      titleAmount = paramsDefault + (_checked*quantity);
+    });
+  }
+
+  void _decreaseQty() {
+    setState(() {
+      quantity--;
+      titleAmount = paramsDefault + (_checked*quantity);
+      // print(quantity);
+    });
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      child: Scaffold(
+        backgroundColor: Colors.white,
+        appBar: AppBar(
```

Comment:
There is CommonAppBar , we should re-use it and modify it to match with the design if needed but let's be consistent here.